client/my-sites/plans-features-main/hooks/use-generate-action-hook.tsx

```diff
@@ -423,7 +423,13 @@ function getLoggedInPlansAction( {
 
 	// Downgrade action if the plan is not available for purchase
 	if ( ! availableForPurchase ) {
-		return createLoggedInPlansAction( translate( 'Downgrade', { context: 'verb' } ), 'secondary' );
+		return {
+			primary: {
+				callback: () => {},
+				text: '',
+				status: 'hidden',
+			},
+		};
```

Comment:
I'd say we should move it behind the `plans/self-service-downgrade` feature flag.